src/scripting/lua_game.cc

```diff
@@ -133,6 +136,18 @@ int LuaPlayer::get_name(lua_State* L) {
 	return 1;
 }
 
+/* RST
+   .. attribute:: number
+
+         (RO) The number of this Player on the map.
```

Comment:
A look in the doc would have saved you some typing ;) This is already a property of Player's parent class so you can delete this function again.
https://www.widelands.org/documentation/autogen_wl_bases/#playerbase